[DM-3891] [LWM2M] Change log for improved snychronization for decoder events (#2381)
SoftwareAG/c8y-docs: content/change-logs/device-management/cumulocity-10-18-540-190-LWM2M-improved-decoder-event-locking.md

---
date:
title: Enhancement on LWM2M decoder event execution
product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: DM-3891
version: 10.18.540.190
---

> Improved the synchronization mechanism to prevent duplicate external decoder events processing by the LWM2M agent.

Reviewer: Either "Improves" or "Has improved"

Author: done

> Improves the synchronization mechanism to prevent duplicate external decoder events processing by the LWM2M agent.

Reviewer 2: ```suggestion
The synchronization mechanism has been improved to prevent duplicate external decoder events processing by the LWM2M agent.
```

> The synchronization mechanism has been improved to prevent duplicate external decoder events processing by the LWM2M agent.

Reviewer 3: ```suggestion
The synchronization mechanism has been improved to prevent the LwM2M Agent from processing duplicate external decoder events.
```

> The synchronization mechanism has been improved to prevent the LwM2M Agent from processing duplicate external decoder events.

Reviewer 2: ```suggestion
The synchronization mechanism has been improved to prevent the LwM2M agent from processing duplicate external decoder events.
```

> The synchronization mechanism has been improved to prevent the LwM2M agent from processing duplicate external decoder events.

Reviewer 2: ```suggestion
The synchronization mechanism has been improved to prevent the LWM2M agent from processing duplicate external decoder events.
```
